exercises/diamond/test/Tests.hs

```diff
@@ -3,15 +3,21 @@
 import Data.Foldable     (for_)
 import Test.Hspec        (Spec, describe, it, shouldBe)
 import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+import Test.QuickCheck   (Gen, oneof, forAll)
+import Data.List         ((\\))
+import Data.Maybe        (isNothing)
 import Diamond (diamond)
 
 main :: IO ()
 main = hspecWith defaultConfig {configFastFail = True} specs
 
 specs :: Spec
-specs = describe "diamond" $ for_ cases test
+specs = describe "diamond" $ do
+  it "non-Alpha characters should produce `Nothing`" $
+    forAll genNonAlphaChars $
```

Comment:
I would probably not go with a plural name; it is a generator of a single non-alpha character, so either `nonAlphaCharGen` or `genNonAlphaChar`, whichever convention is picked.

```suggestion
    forAll genNonAlphaChar $
```